oioioi/participants/admin.py

```diff
@@ -196,7 +196,6 @@ def extend_round(self, request, queryset):
 
     extend_round.short_description = _("Extend round")
 
-
```

Comment:
please remove this change, according to PEP, classes should be separated by two empty lines: https://pymbook.readthedocs.io/en/latest/pep8.html#blank-lines